src/containers/Cfe/index.tsx

```diff
@@ -20,6 +20,8 @@ import { getPropsForVolumeViewer, getViewerHeader, VolumeViewerProps } from "./s
 const { Content, Sider } = Layout;
 
 import styles from "./style.css";
+import { Header } from "antd/es/layout/layout";
+import { PALETTE } from "../../constants";
```

Comment:
another import sorting nit: the antd import can go to the top section, the one from constants could move one block up to its component/redux friends